src/fileLoaders.py

```diff
@@ -81,6 +81,8 @@ def getRawRepoUri(self):
 
     def getTextFor(self, fileItem):
         raw_query_uri = fileItem['download_url']
+        if '/' in raw_query_uri:
```

Comment:
It might be more compact to write:

`raw_query_uri = raw_query_uri.strip('/')`

Also, is this check really necessary? `fileItem` is always an element generated by `fetchFiles`, so as long as `fetchFiles` is behaving properly, this check is not necessary.

Reply:
Sorry @pasqLisena -- I had not fully understood the reason for this change, I was not aware of the slight change in github API.